### ISSUE TITLE
Command: "termius push" 

### ISSUE THREAD

**Issue by ART-008** [Crystalnix/termius-cli#84]
Password:
{"pfrule_set":[{"remote_port":["This field may not be null."]}]}

**Comment by Maxbey** [comment]
Hello, lucien.
How did you create these identities? Did you use import from local ssh config?

**Comment by lucien144** [comment]
@Maxbey Yes, correct. I used import from my local ssh config and nothing has been imported.

**Comment by amanbolat** [comment]
I have same problem.

**Comment by gusutabopb** [comment]
Same problem.
By the way, I use a lot of `*`, meaning that each entry in the `ssh_config` IS NOT independent from each other. I guess the CLI is having some trouble parsing that info.

**Comment by lucien144** [comment]
Would be good to mention my ssh config's structure too - each record is in this format:

```
Host company/dallas/user
	Hostname 1.2.3.4
	User user
	IdentityFile ~/.ssh/id_rsa
```

Then I connect like this:
```sh
ssh company/dallas/user
```

Any plans to fix that? As I said previously - this was the main reason for me to subscribe, so I'm bit disappointed it does not work.

Thanks.
